cmd/rook/main.go

```diff
@@ -27,9 +27,10 @@ import (
 	"github.com/spf13/cobra"
 	"github.com/spf13/pflag"
 
-	"github.com/rook/rook/pkg/ceph/mon"
-	"github.com/rook/rook/pkg/ceph/osd"
+	apiver "github.com/rook/rook/pkg/apis/rook.io/v1alpha1"
```

Comment:
is this a standard to create a package of the version? It seems odd.  So when we promote Rook to beta, there would be quite a bit of changes to update the references.

Reply:
yes, this is the kubernetes pattern. The version of the resources is very specific and will require code churn when we declare beta or v1. It wouldn't be my first choice, but it is the k8s way and is required by the code generation